pallets/delegation/src/lib.rs

```diff
@@ -625,6 +628,62 @@ pub mod pallet {
 			// Add worst case reads from `is_delegating`
 			Ok(Some(<T as Config>::WeightInfo::remove_delegation(removal_checks)).into())
 		}
+
+		/// Reclaim back the deposit for a delegation node (potentially a root
```

Comment:
I think this is superfluous.

You always reclaim something "back". There is no forward reclaiming. This would probably be claiming then.
Further down you write `Returns the delegation deposit back`. Same there. Returning is always back never forward.

```suggestion
		/// Reclaim the deposit for a delegation node (potentially a root
```

Reply:
https://github.com/KILTprotocol/mashnet-node/pull/295/commits/cb08cd6d8c074cc1ce722b8b4c6ec0d62502e034